eventmesh-connectors/eventmesh-connector-http/src/main/java/org/apache/eventmesh/connector/http/sink/handle/RetryHttpSinkHandler.java

```diff
@@ -0,0 +1,214 @@
+                    HttpExportRecord exportRecord = covertToExportRecord(e, e.getResult(), e.getException(), url, firstTryId);
+                    // add the data to the queue
+                    ((WebhookHttpSinkHandler) sinkHandler).addDataToQueue(exportRecord);
+                }
+            })
+            .onRetry(e -> {
+                if (log.isDebugEnabled()) {
+                    log.warn("Retrying the request to {} for the {} time. HttpConnectRecord= {}", url, e.getAttemptCount(), httpConnectRecord);
+                } else {
+                    log.warn("Retrying the request to {} for the {} time.", url, e.getAttemptCount());
+                }
+                if (connectorConfig.getWebhookConfig().isActivate()) {
+                    HttpExportRecord exportRecord = covertToExportRecord(e, e.getLastResult(), e.getLastException(), url, firstTryId);
+                    ((WebhookHttpSinkHandler) sinkHandler).addDataToQueue(exportRecord);
+                }
+            })
+            .onFailure(e -> {
+                if (log.isDebugEnabled()) {
+                    log.error("Failed to send the request to {} after {} attempts. HttpConnectRecord= {}", url, e.getAttemptCount(),
+                        httpConnectRecord, e.getException());
+                } else {
+                    log.error("Failed to send the request to {} after {} attempts.", url, e.getAttemptCount(), e.getException());
+                }
+                if (connectorConfig.getWebhookConfig().isActivate()) {
+                    HttpExportRecord exportRecord = covertToExportRecord(e, e.getResult(), e.getException(), url, firstTryId);
+                    ((WebhookHttpSinkHandler) sinkHandler).addDataToQueue(exportRecord);
+                }
+            }).build();
+
+        // Handle the HttpConnectRecord with retry
+        Failsafe.with(retryPolicy)
+            .getStageAsync(() -> sinkHandler.deliver(url, httpConnectRecord).toCompletionStage());
+
+        return null;
+    }
+
+    /**
+     * Converts the ExecutionCompletedEvent to an HttpExportRecord.
+     *
+     * @param event      the ExecutionCompletedEvent to convert
+     * @param response   the response of the request, may be null
+     * @param e          the exception thrown during the request, may be null
+     * @param url        the URL the request was sent to
+     * @param firstTryId the UUID of the first try
+     * @return the converted HttpExportRecord
+     */
+    private HttpExportRecord covertToExportRecord(ExecutionEvent event, HttpResponse<Buffer> response, Throwable e, URI url, String firstTryId) {
+        HttpExportMetadataBuilder builder = HttpExportMetadata.builder()
+            .url(url.toString())
+            .receivedTime(LocalDateTime.now())
+            .retryNum(event.getAttemptCount() - 1);
+
+        if (event.getAttemptCount() == 1) {
+            builder.retriedBy(null)
+                .uuid(firstTryId);
```

Comment:
The naming of `firstTryId` is easy to be misunderstood with `firstRetryId`.

Reply:
The delivery of `firstTryId` event is actually the first delivery of this event, it's not at the retry stage yet, right? So I think `deliveryId`, `eventId` or simply `id` is fine.